javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+const settingTheHOneInterval = document.querySelector('.countdown-display');
+
+let zero = 4;
+
+function myNumber() {
+  if (zero === 0) {
+    settingTheHOneInterval.textContent = '~earth bellllllooooowwwww us~';
+    settingTheHOneInterval.classList.add('upper');
+  } else {
+    settingTheHOneInterval.textContent = zero;
+    zero--;
+  }
+}
+
+setInterval(myNumber, 1000);
```

Comment:
Need to assign this to a variable so you can keep a reference of the ID to clear this out